content/bn/docs/concepts/extend-kubernetes/_index.md

```diff
@@ -0,0 +1,332 @@
+API সংজ্ঞায়িত করতে চান এবং কুবারনেটিস টুলস যেমন `kubectl` ব্যবহার করে সেগুলো 
+পরিচালনা করতে চান তাহলে কুবারনেটিসে একটি কাস্টম রিসোর্স যোগ করার কথা বিবেচনা করুন।
+
+কাস্টম রিসোর্স সম্পর্কে আরও জানতে, 
+[কাস্টম রিসোর্স](/bn/docs/concepts/extend-kubernetes/api-extension/custom-resources/) কনসেপ্ট গাইড দেখুন।
+
+### API এগ্রিগেশন লেয়ার(aggregation layer)
+
+আপনি কুবারনেটিস [API এগ্রিগেশন লেয়ার](/bn/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/) ব্যবহার করতে পারেন 
+কুবারনেটিস API-কে [মেট্রিক্সের](/bn/docs/tasks/debug/debug-cluster/resource-metrics-pipeline/) মতো অতিরিক্ত সার্ভিসের সাথে একীভূত করতে।
+
+### অটোমেশনের সাথে নতুন API-এর সমন্বয়
+
+একটি কাস্টম রিসোর্স API এবং একটি কন্ট্রোল লুপের সংমিশ্রণকে 
+{{< glossary_tooltip term_id="controller" text="কন্ট্রোলার" >}} প্যাটার্ন বলা হয়। যদি আপনার 
+কন্ট্রোলার একটি কাঙ্ক্ষিত অবস্থার উপর ভিত্তি করে অবকাঠামো স্থাপনকারী মানব অপারেটরের স্থান নেয়, 
+তাহলে কন্ট্রোলারও {{< glossary_tooltip text="অপারেটর প্যাটার্ন" term_id="operator-pattern" >}} অনুসরণ করতে পারে।
+অপারেটর প্যাটার্ন নির্দিষ্ট অ্যাপ্লিকেশন পরিচালনা করতে ব্যবহৃত হয়; সাধারণত, এগুলো হলো এমন অ্যাপ্লিকেশন 
+যা অবস্থা বজায় রাখে এবং সেগুলোকে কীভাবে পরিচালনা করা হয় তার যত্নের প্রয়োজন হয়৷
+
+আপনি আপনার নিজস্ব কাস্টম API এবং কন্ট্রোল লুপগুলোও তৈরি করতে পারেন যা অন্যান্য রিসোর্সগুলো পরিচালনা করতে পারে, 
+যেমন স্টোরেজ, বা পলিসিগুলো সংজ্ঞায়িত করতে (যেমন একটি অ্যাক্সেস কন্ট্রোল রেস্ট্রিকশন)।
+
+### বিল্ট-ইন রিসোর্স পরিবর্তন
+
+আপনি যখন কাস্টম রিসোর্স যোগ করে কুবারনেটিস API প্রসারিত করেন, তখন যোগ করা রিসোর্স সবসময় 
+একটি নতুন API গ্রুপে পড়ে। আপনি বিদ্যমান API গ্রুপগুলোকে প্রতিস্থাপন বা পরিবর্তন করতে পারবেন না ৷
+একটি API যোগ করলে আপনাকে বিদ্যমান API-এর আচরণকে সরাসরি প্রভাবিত করতে দেওয়া না (যেমন পড), 
+যেখানে _API অ্যাক্সেস এক্সটেনশানগুলো_ করে।
+
+## API অ্যাক্সেস এক্সটেনশন
+
+যখন একটি অনুরোধ কুবারনেটিস API সার্ভারে পৌঁছায়, এটি প্রথমে _অথেন্টিকেটেড(authenticated)_ করা হয়, 
+তারপর _অনুমোদিত(authorized)_ হয় এবং তারপরে আসে বিভিন্ন ধরণের _অ্যাডমিশন কন্ট্রোলের(admission control)_ 
+বিষয় (কিছু অনুরোধ প্রকৃতপক্ষে অথেন্টিকেটেড(authenticated) নয়, এবং স্পেশাল ট্রিটমেন্ট পান)।
+এই প্রবাহ সম্পর্কে আরও জানতে 
+[কুবারনেটিস API-এ অ্যাক্সেস কন্ট্রোল করা](/bn/docs/concepts/security/controlling-access/) দেখুন।
+
+কুবারনেটিস অথেন্টিকেশন/অনুমোদন প্রবাহের প্রতিটি ধাপ এক্সটেনশন পয়েন্ট অফার করে।
+
+### অথেন্টিকেশন(Authentication)
+
+ক্লায়েন্ট অনুরোধ করার জন্য একটি ব্যবহারকারীর নামের সমস্ত অনুরোধে 
+[অথেন্টিকেশন](/bn/docs/reference/access-authn-authz/authentication/)  শিরোনাম বা সার্টিফিকেট যুক্ত করে।
+
+কুবারনেটিস এর বেশ কয়েকটি বিল্ট-ইন অথেন্টিকেশন পদ্ধতি রয়েছে যা এটি সাপোর্ট করে। 
+এটি একটি অথেন্টিকেটিং প্রক্সির পিছনেও বসতে পারে এবং এটি একটি `অনুমোদন(Authorization)` টোকেন পাঠাতে পারে যাচাইয়ের জন্য: 
+শিরোনাম থেকে  একটি রিমোট সার্ভিসে (একটি [অথেন্টিকেশন webhook](/bn/docs/reference/access-authn-authz/authentication/#webhook-token-authentication)) 
+যদি সেগুলো আপনার প্রয়োজনগুলো পূরণ না করে৷
+
+### অনুমোদন(Authorization)
```

Comment:
```suggestion
### অনুমোদন(Authorization) {#authorization}
```
?

Reply:
Just to confirm, {#authorization} isn't in the en file. Should I add it in bn ?

![image](https://github.com/kubernetes/website/assets/155321064/3dc1c8d7-d61a-4267-b964-237ada5aa43a)